applications/MeshingApplication/custom_utilities/parmmg/pmmg_utilities.cpp

```diff
@@ -1324,7 +1324,6 @@ void ParMmgUtilities<TPMMGLibrary>::WriteMeshDataToModelPart(
 
         std::unordered_map<IndexType, std::vector<IndexType>> pi_to_vector_node;
         //get nodes we need from other ranks
-        IndexType rank =  DataCommunicator::GetDefault().Rank();
```

Comment:
no clue how this works, this is declared above

@marcnunezc can you take a look please?

Reply:
probably a leftover when first working on this